layouts/PageLayout.tsx

```diff
@@ -25,9 +25,7 @@ class Layout extends React.Component {
   render() {
     return (
       <div style={layoutStyle}>
-        <MainSiteHeader links={this.props.mappedCategories} />
```

Comment:
Was there a specific reason the MainSiteHeader and Footer components were removed from the layout file?